resources/developer-guides/daemon-rpc.md

```diff
@@ -2181,7 +2181,7 @@ Outputs:
 Example:
 
 ```
-$ curl -X POST http://127.0.0.1:18081/get_limit -H 'Content-Type: application/json'
+$ curl http://127.0.0.1:18081/get_limit -H 'Content-Type: application/json'
```

Comment:
The `-X POST` was not unnecessary here as there is no `-d` flag, however I don't think this should be a POST anyway since we just get data. (I tested with GET and POST, you get the same response)